process/transaction/metaProcess.go

```diff
@@ -88,12 +95,23 @@ func (txProc *metaTxProcessor) ProcessTransaction(tx *transaction.Transaction) e
 
 	switch txType {
 	case process.SCDeployment:
-		return txProc.processSCDeployment(tx, adrSrc)
+		err = txProc.processSCDeployment(tx, adrSrc)
+		return err
 	case process.SCInvoking:
-		return txProc.processSCInvoking(tx, adrSrc, adrDst)
+		err = txProc.processSCInvoking(tx, adrSrc, adrDst)
```

Comment:
Why not directly: return txProc.processSCInvoking(tx, adrSrc, adrDst) ?

Reply:
done